core/merginapi.cpp

```diff
@@ -1364,6 +1364,11 @@ void MerginApi::checkMerginVersion( QString apiVersion, bool serverSupportsSubsc
     {
       setApiVersionStatus( MerginApiStatus::INCOMPATIBLE );
     }
+
```

Comment:
Can you add a comment here please to describe why we do this?

Reply:
added :)